packages/contracts-bedrock/src/L2/L2ToL2CrossDomainMessenger.sol

```diff
@@ -64,8 +64,8 @@ contract L2ToL2CrossDomainMessenger is ISemver, TransientReentrancyAware {
     uint16 public constant messageVersion = uint16(0);
 
     /// @notice Semantic version.
-    /// @custom:semver 1.1.0
-    string public constant version = "1.1.0";
+    /// @custom:semver 1.1.1
```

Comment:
Per the [semver guide](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/book/src/policies/versioning.md#semver-rules) this should be a major bump but given its prerelease software, it makes more sense to make it a minor bump as it doesn't make sense to be a patch

Reply:
ah yea good call, minor makes sense